### PR TITLE
Feature/java

### PR DESCRIPTION
**Pull request by lfchris**

### CODE REVIEW

File: roles/java/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+This role installs [openjdk](https://openjdk.java.net).
+
+FQCN: linuxfabrik.lfops.java
+
+Tested on
+
+* Debian
+* RHEL 7 (and compatible)
+* RHEL 8 (and compatible)
+* Fedora
```
Comment (NavidSassan) [inline roles/java/README.md:12]: are those true? else remove

File: roles/java/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+- name: 'yum install java'
+  yum:
```
Comment (NavidSassan) [inline roles/java/tasks/main.yml:2]: use the fqcn, and `package` instead of `yum`

File: roles/java/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    name:
+      - 'java'
+    state: 'present'
+  when:
+    - 'ansible_facts["os_family"] == "RedHat" and ansible_facts["distribution_major_version"] in ["7", "8"]
+      or ansible_facts["distribution"] == "Fedora"'
```
Comment (NavidSassan) [inline roles/java/tasks/main.yml:8]: do not use two sections with `when`-statements, use a variable (`java__package`) for the package instead and include it in the vars files (see platform-variables in the CONTRIBUTING).

**Review by NavidSassan — CHANGES_REQUESTED** [review]

File: roles/java/tasks/main.yml (hunk 2 above)
Comment (lfchris) [reply roles/java/tasks/main.yml:2]: done

**Review by NavidSassan — APPROVED** [review]

**Pull request merged**